docs.kosli.com/content/getting_started/install.md

```diff
@@ -11,6 +11,28 @@ Kosli CLI can be installed from package managers,
 by Curling pre-built binaries, or can be used from the distributed Docker images.
 {{< tabs "installKosli" >}}
 
+{{< tab "Script" >}}
+You can download the correct Kosli CLI for your platform, given that you can run bash on it, by invoking this one-line script:
```

Comment:
Don't know if its important or not...
You mention bash, but the command is piped into `sh` 
and the shebang in the script is `bin/sh`

Reply:
So changing it to 
`You can download the correct Kosli CLI for your platform, given that you can run shell scripts on it, by invoking this one-line script:`
Would be more correct, right?